src/App.jsx

```diff
@@ -14,57 +15,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
```

Comment:
Recommend creating a separate function for clearing the selection `clearSelection` for better readability.
```
 const clearSelection = () => setSelectedGood('');
```
```
onClick={clearSelection}
```